packages/web3-shared/evm/types/index.ts

```diff
@@ -162,8 +162,11 @@ export interface ERC721ContractDetailed extends ERC721Token {
 export interface ERC721TokenInfo {
     name?: string
     description?: string
+    tokenURI?: string
     mediaUrl?: string
     owner?: string
+    // loading tokenURI
+    loading?: boolean
```

Comment:
Don't add a status probe here.

Reply:
I'd like to rename it to `hasTokenDetailed`. Since we need to know whether we have fetched the data from `tokenURI`